_reports/saksham.md

```diff
@@ -0,0 +1,266 @@
+---
+layout: report
+permalink: /GSoC/2018/StatusReport/SakshamBansal
+student: Saksham Bansal
+organisation: coala
+organisation_link: https://coala.io
+report_year: 2018
+project: Improve installation of coala and coala-bears
+project_link: https://summerofcode.withgoogle.com/projects/#4642118669172736
+tarball: https://rawgit.com/anctartica/GSoC-Report/master/work.tar.gz
+sha256sum: f413963583c7e18a5ccb42f21405290edcf79a0f8c61106ff21671efe0b98238
+mentors: >
+ [Muhammad Kaisar Arkhan](https://github.com/yukiisbored),
+ [Matt Bodenhamer](https://github.com/mbodenhamer),
+ [Andrey Safonov](https://github.com/xferra)
+phase:
+ - Bonding: https://gitlab.com/coala/GSoC/GSoC-2018/milestones/10
```

Comment:
indentation; use yamllint

Reply:
I don't get anything from yamllint on this line